python/src/scipp/compat/mantid.py

```diff
@@ -204,6 +204,14 @@ def _to_spherical(pos, output):
     sc.atan2(sc.geometry.y(pos), sc.geometry.x(pos), output["p"].data)
 
 
+def to_quat(mat):
+    w = np.sqrt(np.maximum(0.0, 1.0 + mat[0][0] + mat[1][1] + mat[2][2])) / 2.0
+    x = np.sqrt(np.maximum(0.0, 1.0 + mat[0][0] - mat[1][1] - mat[2][2])) / 2.0
+    y = np.sqrt(np.maximum(0.0, 1.0 - mat[0][0] + mat[1][1] - mat[2][2])) / 2.0
+    z = np.sqrt(np.maximum(0.0, 1.0 - mat[0][0] - mat[1][1] + mat[2][2])) / 2.0
+    return sc.Variable(value=sc.Quat([w, x, y, z]))
```

Comment:
careful with the order in the constructor. I think we have `Quat([x, y, z, w])` when you give the whole list.